driver/pace/driver/initialization.py

```diff
@@ -125,26 +122,18 @@ class RestartConfig(Initializer):
     Configuration for restart initialization.
     """
 
-    path: str
-
-    @property
-    def start_time(self) -> datetime:
-        return datetime(2000, 1, 1)
+    path: str = "."
+    start_time: datetime = datetime(2000, 1, 1)
 
     def get_driver_state(
         self,
         quantity_factory: pace.util.QuantityFactory,
         communicator: pace.util.CubedSphereCommunicator,
     ) -> DriverState:
-        metric_terms = pace.util.grid.MetricTerms(
-            quantity_factory=quantity_factory, communicator=communicator
-        )
-        state = pace.util.open_restart(
-            dirname=self.path,
-            communicator=communicator,
-            quantity_factory=quantity_factory,
+        state = _restart_driver_state(
+            self.path, str(communicator.rank), quantity_factory, communicator
```

Comment:
Optional: change the argument type to `int` and cast to `str` inside the `_restart_driver_state` routine.